docs/howto/how-to-contribute.md

```diff
@@ -17,9 +17,9 @@ This section explains the steps to create a pull request (PR).
 1. Create an issue
 
    Maintainers will accept your contribution only when it is well aligned with the 
-   [roadmap](./overview/roadmap.md) and design principles of **ONE**. So, it is optional, but 
-   recommended for contributors to create an issue and have a discussion with maintainers before 
-   writing code.
+   [roadmap](https://github.com/Samsung/ONE/blob/master/docs/overview/roadmap.md) and design
```

Comment:
can you please check if `../overview/roadmap.md` works?

Reply:
```suggestion
   [roadmap](../overview/roadmap.md) and design
```

@seanshpark It works. Thank you for your recommendation.